package.json

```diff
@@ -1,7 +1,9 @@
 {
   "name": "morphir-elm",
-  "version": "2.68.0",
+  "version": "2.69.0",
```

Comment:
I'm a bit confused about this. You published this from your fork? The instructions say that you should do this from the finos org (upstream): https://github.com/finos/morphir-elm/blob/main/docs/contribution-guide/publishing.md#publishing-the-npm-package

So the workflow is: merge your changes through a PR, then publish the NPM package from upstream using a separate clone.